lib/teslamate/vehicles/vehicle/summary.ex

```diff
@@ -87,6 +87,7 @@ defmodule TeslaMate.Vehicles.Vehicle.Summary do
       charge_current_request_max: charge(vehicle, :charge_current_request_max),
       charge_energy_added: charge(vehicle, :charge_energy_added),
       charge_limit_soc: charge(vehicle, :charge_limit_soc),
+      charge_rate: charge(vehicle, :charge_rate),
```

Comment:
adding a test to one of the following would be good:
 
- [test/teslamate/vehicles/vehicle_sync_test.exs](https://github.com/teslamate-org/teslamate/blob/master/test/teslamate/vehicles/vehicle_sync_test.exs)
- test/teslamate/vehicles/vehicle/charging_sync_test.exs
- test/teslamate/vehicles/vehicle/charging_test.exs

Reply:
Fields added to the `Summary` module are typically published to MQTT. Have you verified that the new field is pushed correctly in various vehicle states such as driving, charging, and idling?